src/styles/index.scss

```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  padding: 0 $boxPadding;
+  width: $boxWidth;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @for $i from 1 through 31 {
+    :nth-child(#{$i})::before {
+      content: '#{$i}';
+      font-size: 30px;
+      font-family: Arial, sans-serif;
+    }
+  }
+  :nth-child(n) {
+    display: none;
+  }
+}
+
+@for $i from 28 through 31 {
+  .calendar--month-length-#{$i} {
+    :nth-child(-n + #{$i}) {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: $calendarDaySize;
+      height: $calendarDaySize;
+      box-sizing: border-box;
+      border: 1px solid black;
+      background-color: #eee;
```

Comment:
Consider moving repeating styles to a basic class instead of generating them for every element in the cycle